### ISSUE TITLE
Transformations in basis set

### ISSUE THREAD

**Issue by jslefche** [jslefche/piecewiseSEM#50]
Should consider removing or somehow handling in a way that does not duplicate if some vars are transformed (predictors) and others are not (responses)

**Comment by jslefche** [comment]
Think I got this in latest update (v 1.1) but should continue to check to make sure it didn't break anything...

**Comment by jslefche** [comment]
Hmm can't figure out why but this is back. Currently switching this off because the error occurs in re-applying transformations to the basis set in `sem.basis.set`

**Comment by jslefche** [comment]
Reproducible example: 

```
set.seed(1)

data = data.frame(
  x1 = runif(50),
  x2 = runif(50),
  x3 = runif(50),
  x4 = runif(50)
)

modelList = list(
  lm(x2 ~ x1, data),
  lm(scale(x3) ~ log10(x2), data),
  lm(x4 ~ x3, data)
)

sem.basis.set(modelList)

sem.fit(modelList, data) # conditional = T)
```

The basis set should be:

```
$`1`
[1] "x1"        "scale(x3)" "log10(x2)"

$`2`
[1] "x1" "x4" "x3"

$`3`
[1] "log10(x2)" "x4"        "x1"        "x3"    
```

**Comment by jslefche** [comment]
Ok looks like fixed for now but keeping open just in case...

**Comment by jslefche** [comment]
A new way to index transformed variables:

```
data = data.frame(
  y = runif(50),
  x1 = runif(50),
  x2 = runif(50)
)

modelList = list(lm(y ~ log(x1 + 12) + sqrt(x2), data))

# Get transformed
lapply(modelList, function(i) as.character(attr(terms(i), "variables"))[-1] )

# Get untransformed
lapply(modelList, function(i) all.vars(formula(i)) )
```

**Comment by jslefche** [comment]
Fixed in 2.0
